### ISSUE TITLE
Security shuttle

### ISSUE THREAD

**Issue by Piras314** [Goob-Station/Goob-Station#633]
self-explanatory, requires security to have a dock

the shuttle should be armed and protected with turrets that shoot anybody without a mindshield

**Comment by Piras314** [comment]
a big reason for this is to combat the reclaimer and cargo shuttle being essentially lawless and to add more shuttle gameplay into the mix, along with #632

**Comment by HankKitts** [comment]
I agree but the mindshield sounds like it'd just cause issues. But Security should have at least a basic one, maybe instead of turrents it could have an external walkway with railings, so a Secoff with a gun could stand on them and shoot from the shuttle? 
![5e6509c0bf0ea image](https://github.com/user-attachments/assets/32a9a865-b060-4c38-95ca-74f2275db3f1)
